src/Realm.C

```diff
@@ -974,7 +974,13 @@ Realm::setup_post_processing_algorithms()
 
     // call through to the Eqsys
     if ( theType == "surface" ) {
-      equationSystems_.register_surface_pp_algorithm(theData);
+      for (size_t ibc = 0; ibc < boundaryConditions_.size(); ++ibc) {
+        BoundaryCondition& bc = *boundaryConditions_[ibc];
+        std::string name = physics_part_name(bc.targetName_);
+        if (bc.theBcType_ == WALL_BC) { 
+          equationSystems_.register_surface_pp_algorithm(theData, *reinterpret_cast<const WallBoundaryConditionData *>(&bc));
+        }
+      }
```

Comment:
Recommend shifting this closer to where the SST wall function algorithms are registered.

Reply:
Where do you mean by that? Are you saying to move it outside of setup_post_processing_algorithms?

Reply:
Looking at what is being done in `SurfaceForceMoment*Alg`, that functionality is already available in `SurfaceForceMomentWallFuncAlg`.. if you switch to using that then you don't need to change the existing `register_surface_pp_algorithm` logic.